src/config.rs

```diff
@@ -1171,23 +1200,23 @@ impl Default for StorageReadPoolConfig {
             high_concurrency: DEFAULT_STORAGE_READPOOL_CONCURRENCY,
             normal_concurrency: DEFAULT_STORAGE_READPOOL_CONCURRENCY,
             low_concurrency: DEFAULT_STORAGE_READPOOL_CONCURRENCY,
-            max_tasks_per_worker_high: readpool::config::DEFAULT_MAX_TASKS_PER_WORKER,
-            max_tasks_per_worker_normal: readpool::config::DEFAULT_MAX_TASKS_PER_WORKER,
-            max_tasks_per_worker_low: readpool::config::DEFAULT_MAX_TASKS_PER_WORKER,
-            stack_size: ReadableSize::mb(readpool::config::DEFAULT_STACK_SIZE_MB),
+            max_tasks_per_worker_high: DEFAULT_READPOOL_MAX_TASKS_PER_WORKER,
+            max_tasks_per_worker_normal: DEFAULT_READPOOL_MAX_TASKS_PER_WORKER,
+            max_tasks_per_worker_low: DEFAULT_READPOOL_MAX_TASKS_PER_WORKER,
+            stack_size: ReadableSize::mb(DEFAULT_READPOOL_STACK_SIZE_MB),
         }
     }
 }
 
 const DEFAULT_COPROCESSOR_READPOOL_CONCURRENCY: usize = 8;
 
 readpool_config!(
-    CoprocessorReadPoolConfig,
+    CoprReadPoolConfig,
```

Comment:
Copr or Cop?

Reply:
Both are ok. I prefer Copr now because "cop" is a valid word..XD